pkg/project/auth/watch.go

```diff
@@ -149,7 +149,7 @@ func (w *userProjectWatcher) GroupMembershipChanged(namespaceName string, users,
 
 		event := watch.Event{
 			Type:   watch.Added,
-			Object: projectutil.ConvertNamespace(namespace),
+			Object: projectutil.ConvertNamespaceExternalOrDie(namespace),
```

Comment:
I'm not sure we want to die here, @deads2k ?

Reply:
I could switch this to return an error and use `utilruntime.HandleError` instead